primitives/src/proxy_type.rs

```diff
@@ -36,6 +37,11 @@ pub enum ProxyType {
     CancelProxy,
     Governance,
     Staking,
+    CreateMarket,
```

Comment:
Can a proxy only have one type? Meaning if you want to grant a user the ability to create markets and report outcomes but not provide liquidity or trade you have to have two proxy accounts of the individual types?

Reply:
https://wiki.polkadot.network/docs/learn-proxies

[You can have multiple proxies for one account.](https://github.com/paritytech/substrate/blob/06689cbbdad85cf9fcab87a26b0c3b8e93b375ec/frame/proxy/src/lib.rs#L645-L651)

Means: You add a "CreateMarket" proxy for account A and then you can add a "ProvideLiquidity" proxy for the same account A.

Now account A can create markets and do liquidity provision.

Reply:
So I guess when the proxy transaction is wrapped using polkadotjs we have to specify the correct proxy type for the extrinsic like this?
```ts
sdk.api.tx.proxy.proxy(proxy?.address, "CreateMarket", createMarketExtrinsic);
sdk.api.tx.proxy.proxy(proxy?.address, "ReportOutcome", reportExtrinsic);
```